Samples/BackgroundBlur/BackgroundBlur/common/critsec.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
```

Comment:
[It's always better to include what you use](https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md)